homeassistant/components/reolink/entity.py

```diff
@@ -52,17 +54,16 @@ def available(self) -> bool:
         return self._host.api.session_active and super().available
 
 
-class ReolinkCoordinatorEntity(ReolinkBaseCoordinatorEntity):
+class ReolinkCoordinatorEntity(ReolinkBaseCoordinatorEntity[None]):
     """Parent class for Reolink hardware camera entities connected to a channel of the NVR."""
 
     def __init__(
         self,
         reolink_data: ReolinkData,
         channel: int,
-        coordinator: DataUpdateCoordinator | None = None,
```

Comment:
Please leave the option to specify a diffrent coordinator, this may be used in the future.